src/components/UriViewer/UriViewer.js

```diff
@@ -56,28 +66,60 @@ export const UriViewer = _.flow(
         setMetadata(metadata)
       }
     } catch (e) {
-      setLoadingError(await e.json())
+      setLoadingError(await e.text())
```

Comment:
temp change while I figure out why the response coming from azure is xml

Reply:
https://feedback.azure.com/d365community/idea/0646f944-3a25-ec11-b6e6-000d3a4f0f84